basket/news/tasks.py

```diff
@@ -622,8 +638,27 @@ def sfdc_add_update(update_data, user_data=None):
                 sfdc.add(update_data)
 
 
+@et_task
+def send_acoustic_tx_message(email, vendor_id, fields=None):
+    acoustic_tx.send_mail(email, vendor_id, fields)
+
+
+def send_acoustic_tx_messages(data, message_ids):
+    sent = 0
+    for mid in message_ids:
+        vid = AcousticTxEmailMessage.objects.get_vendor_id(mid, data.get("lang", "en"))
```

Comment:
Here you use `en` but in https://github.com/mozmeao/basket/pull/655/files#diff-c21115099033cfff1ad2585abb19c8ca144a4cac5b26bd74a60b5c87fadd908eR380 you use `en-US`. Just checking that both are correct?

Reply:
That's a good catch. Should probably use the full `en-US` here too since that's what'll be in the DB and will be the most efficient query.